pkgs/development/python-modules/astropy-healpix/default.nix

```diff
@@ -36,6 +36,10 @@ buildPythonPackage rec {
     hypothesis
   ];
 
+  disabledTests = [
+    "test_interpolate_bilinear_skycoord"
```

Comment:
This test seems to pass on x86_darwin. Perhaps add a conditional for x86_linux?

Reply:
Is it good now?